qos/qos_rule.go

```diff
@@ -328,10 +310,9 @@ func (ipfRule *IPFilterRule) decodeIpFilterPortInfo(source bool, tag string) err
 			ipfRule.dPort = ports[0]
 		}
 	}
-	return nil
 }
 
-func (ipfRule *IPFilterRule) decodeIpFilterAddrv4(source bool, tag string) error {
```

Comment:
Why did you remove the `return` value (`error`)? Should I understand that somehow it was unused or irrelevant?

Reply:
there was no condition where it would have returned anything else then nil